test/e2e/data/infrastructure-vsphere-supervisor/main/install-on-bootstrap/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I would prefer to defer this to another PR because the existing patch cannot be used (it applies to legacy clusters. install on bootstrap uses a classy cluster)

Also, this isn't a problem that exists for govimomi templates.
I have opened https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2898 to track this

Reply:
Was this test red for govmomi or is this something new with supervisor?

I assumed current main is stable (or was at least until a few days ago)

Reply:
The test was geen for govimomi, but it failed in the few runs I did for supervisor (not sure if for the lack of resources  or for the fact we were running many conformance test in parallel, which is something we usually don't do)